fuzzer/modules/sql_injection/strategies/lab6_string_interpolation.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import uuid
+
+VULNERABLE_ENDPOINT = 'http://127.0.0.1:4000/secret'
+
+
+def lab6_string_interpolation_test(host, grammar):
```

Comment:
I think the union attack is a general strategy that's used, though I think its more commonly used to select out system meta table information. In this case, since lab6 uses a fake sqllite db, there's nothing for us to connect to and no actual meta table data. Ugh..... In any case, lets call this the `union strategy` so it can be more generic